src/img/arrow-right.svg

```diff
@@ -0,0 +1 @@
+<svg id="icon-arrow" class="icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24.05 36.18"><defs><style>.a{font-size:36px;font-family:Ionicons, Ionicons;}</style></defs><title>arrow-right</title><text class="a" transform="translate(0 31.68)"></text></svg>
```

Comment:
this SVG uses some kind of character rather than vector lines. On my mac is just shows a square box.